variants/FYSETC_S6/PeripheralPins.c

```diff
@@ -0,0 +1,361 @@
+/*
```

Comment:
Please update this file with the latest generated, this will avoid further annoying issue:
https://github.com/stm32duino/Arduino_Tools/tree/master/src/genpinmap/Arduino/STM32F4/STM32F446V(C-E)Tx